### DISCUSSION TITLE
RichEditor with s3

### DISCUSSION THREAD

**Discussion by feliper-botelho** [filamentphp/filament#6844] [Help]
When im using s3 as my default filesystem, it doesnt save my file attachment on content. It doesnt throw any error.
The file uploads to livewire-tmp on my bucket, but doesnt create the file outside it.
If i try  fileAttachmentsDisk('public') it works fine.

someone got the same error?
thanks

**Comment by danharrin** [comment]
Sorry, I dont know how to help without errors

**Reply by feliper-botelho** [reply]
thats ok, i gonna use public disk just for richEditor
thanks
